### PR TITLE
Omitting deleted volumes while collection global cluster details

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Tendrl-bugid: Tendrl/gluster-integration#465

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n @nnDarshan please review

**Comment by GowthamShanmugam** [comment]
yes tested

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 1]
```diff
@@ -234,7 +234,11 @@ def run(self):
                 # Sync cluster global details
                 if "provisioner/%s" % NS.tendrl_context.integration_id \
                     in NS.node_context.tags:
-                    volumes = NS.gluster.objects.Volume().load_all() or []
+                    all_volumes = NS.gluster.objects.Volume().load_all() or []
+                    volumes = []
+                    for volume in all_volumes:
+                        if not eval(volume.deleted):
```
Comment (r0h4n) [inline tendrl/gluster_integration/sds_sync/__init__.py]: The **eval** function lets a python program run python code within itself. It is a security risk to use eval.

Check for volume.delete.lower() == "true"

Comment (GowthamShanmugam) [reply tendrl/gluster_integration/sds_sync/__init__.py]: ok i will change this

Comment (shtripat) [reply tendrl/gluster_integration/sds_sync/__init__.py]: Also only to check the value of `volume.deleted` flag, why we need eval?

**Comment by GowthamShanmugam** [comment]
@r0h4n modifier and verified

**Pull request merged**